CONTRIBUTING.md

```diff
@@ -118,6 +118,7 @@ In case of spurious cache-related issues with Gradle build, try one of the follo
 * `./gradlew clean` and re-run the failing `./gradlew` command with `--no-build-cache`
 * remove .gradle/ directory in the project directory
 * remove ~/.gradle/caches/ (or even the entire ~/.gradle/) directory
+* reinstall gradle AND remove entire ~/.gradle/ directory
```

Comment:
"remove... ~/.gradle/" - This one is included in the previous line but anyway it can remain

Reply:
But you could use "the" before entire (assuming it's correct grammatically)

Reply:
yes, I had to do both things to fix my issue